lib/instrumentation/core/http-outbound.js

```diff
@@ -205,16 +205,18 @@ function assignOutgoingHeaders(currentHeaders, outboundHeaders) {
 /**
  * Starts the http outbound segment and attaches relevant attributes to the segment/span.
  *
+ * @param {Agent} agent New Relic agent
  * @param {string|object} opts a url string or HTTP request options
  * @param {Function} makeRequest function to make request
  * @param {string} hostname host of outbound request
  * @param {TraceSegment} segment outbound http segment
  * @returns {http.IncomingMessage} request actual http outbound request
  */
-function applySegment(opts, makeRequest, hostname, segment) {
```

Comment:
I believe `segment.transaction.agent` already has the agent.

Reply:
Brilliant, thanks for the info, will check this out